bokeh/models/mappers.py

```diff
@@ -50,17 +50,47 @@ class LinearColorMapper(ColorMapper):
     this are clamped to ``high``.
     """)
 
-    # TODO: (jc) what is the color code for transparent?
-    # TODO: (bev) better docstring
-    reserve_color = Color("#ffffff", help="""
-    Used by Abstract Rendering.
```

Comment:
@bryevdv - do you want me to add it back or deprecate? I have no strong opinion either way.